users/vieting/experiments/switchboard/ctc/feat/baseline_args.py

```diff
@@ -229,7 +235,7 @@ def get_returnn_config(
             "learning_rate_control_relative_error_relative_lr": True,
             "min_learning_rate": 1e-5,
             "newbob_learning_rate_decay": 0.9,
-            "newbob_multi_num_epochs": datasets["train"]["partition_epoch"],
```

Comment:
This change is needed, because the dataset changed to a Multiproc Dataset

Reply:
Yes. MultiProc Dataset is needed for audio perturbation and has no negative effects on other trainings.

Reply:
If you enable it for all trainings, it changes all hashes which is never a good idea. Please only use it when necessary.

Reply:
This change here seems to be wrong, doesn't it?

Reply:
@Max-Ryujin this is still wrong, isn't it?

Reply:
this is now dependent on `use_multi_proc_dataset`